src/plugins/packages/index.js

```diff
@@ -86,3 +95,44 @@ function loadPackages(clientManager) {
 		log.info(`Package ${colors.bold(packageName)} loaded`);
 	});
 }
+
+async function outdated() {
+	if (cache.outdated !== undefined) {
```

Comment:
```suggestion
	if (typeof cache.outdated !== "undefined") {
```

Reply:
I'm not sure this is unnecessary, because we specifically specify it is undefined and then delete it, so when would this be different?